coderd/database/awsiamrds/awsiamrds_test.go

```diff
@@ -33,7 +34,7 @@ func TestDriver(t *testing.T) {
 	sqlDriver, err := awsiamrds.Register(ctx, "postgres")
 	require.NoError(t, err)
 
-	db, err := cli.ConnectToPostgres(ctx, testutil.Logger(t), sqlDriver, url, migrations.Up)
+	db, err := cli.ConnectToPostgres(ctx, testutil.Logger(t), sqlDriver, url, migrations.Up, codersdk.DefaultMaxConns, codersdk.DefaultIdleConns)
```

Comment:
suggestion: Should we also test that max/idle conns is set correctly when using awsiamrds driver? I'm not sure if it will run by default in CI, but it feels like a good potential touch point.

Reply:
Also appears not so feasible